app/services/cfe/base_service.rb

```diff
@@ -0,0 +1,82 @@
+module CFE
+  class BaseService
+    def self.call(submission)
+      new(submission).call
+    end
+
+    def initialize(submission)
+      @submission = submission
+      @conn = Faraday.new(url: cfe_url_host)
+    end
+
+    def call
+      @response = query_cfe_service
+      process_response
+    end
+
+    private
+
+    def cfe_url_host
+      'http://localhost:3001'
+    end
+
+    def cfe_url
+      cfe_url_host + cfe_url_path
```

Comment:
Use `File.join` or `URI.join` here to avoid issues with leading and trailing slashes.